doc/nrf/ug_creating_matter_accessory.rst

```diff
@@ -0,0 +1,267 @@
+Creating Matter Sensor application from Matter Template sample
```

Comment:
I think you can use the title corresponding with the file name: `Creating Matter accessory`. The description below will explain that a sensor device has been selected as an example. Also, add `.. ug_creating_matter_accessory:` before this line, so we can refer to the article in other READMEs.